### PR TITLE
Generate doc for docker toolbox

### PR DESCRIPTION
**Pull request by fperdigon**
Generate doc for docker toolbox. Update the doc examples to work with SCT v3,2.1. Minor fixes to the previous doc. 

solves issue #3

### CODE REVIEW

File: windows/sct-win_docker_toolbox.xlaunch [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0"?>
+<XLaunch xmlns="http://www.straightrunning.com/XmingNotes" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.straightrunning.com/XmingNotes XLaunch.xsd" WindowMode="MultiWindow" ClientMode="StartProgram" Program="lxterminal" ClientStart="OpenSSH" PathToProtocol="C:\\Program Files\\Git\\usr\\bin\\" RemoteHost="192.168.99.100" RemoteUser="sct" ExtraSSH="-p 2222 -Y" Display="0" Clipboard="true"/>
```
Comment (zougloub) [inline windows/sct-win_docker_toolbox.xlaunch:2]: Didn't you want to not use git's SSH?

**Review by zougloub — CHANGES_REQUESTED** [review]
Almost good for pull but:
- PR title can be improved
- Please confirm choice of SSH client on Windows
- Is the remote host of Docker Toolbox always the same 192.168.99.100 ?

File: windows/sct-win_docker_toolbox.xlaunch (hunk 1 above)
Comment (fperdigon) [reply windows/sct-win_docker_toolbox.xlaunch:2]: Yes, you are right. I completely forgot.

**Comment by fperdigon** [comment]
@zougloub The 192.168.99.100 IP is generated by docker toolbox when creates the VM called default, always use the same IP. However, this IP can be changed by the user, let's assume that the user will not change it. :)

**Pull request merged**